packages/DraftEditor/src/LinkControl/Anchor.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { ILinkProps } from "../interfaces";
+
+const Anchor = (props: ILinkProps): JSX.Element => {
+  const { url }: { url: string } = props.contentState
+    .getEntity(props.entityKey)
+    .getData();
+  return (
+    <a rel="nofollow noreferrer" href={url} target="_blank">
+      {props.children}
```

Comment:
Why aren't you doing destructuring of children in the declaration of the component?